docker/common.ps1

```diff
@@ -35,24 +35,24 @@ function GetUbuntuImageTags($repo, $version, $arch) {
     switch ($arch) {
         "x64" { 
             return @(
-                "$($repo):latest",
```

Comment:
don't we need both? it will be latest and latest-lts at the same time?

Reply:
nice catch, we didn't have such a case yet. I just took stuff from 5.2.